setup.py

```diff
@@ -31,6 +31,7 @@
 requirements = [
     'Flask',
     'six',
+    'invenio-base>=0.2.1',
     'invenio-upgrader>=0.1.0',
     'Invenio',
```

Comment:
please remove invenio and add invenio_records, invenio_accounts, invenio_oauth2server